packages/teams-js/CHANGELOG.md

```diff
@@ -1,9 +1,30 @@
 # Change Log - @microsoft/teams-js
 
-This log was last generated on Wed, 01 Nov 2023 18:15:02 GMT and should not be manually modified.
+This log was last generated on Thu, 30 Nov 2023 20:36:13 GMT and should not be manually modified.
 
 <!-- Start content -->
 
+## 2.18.0
+
+Thu, 30 Nov 2023 20:36:13 GMT
+
+### Minor changes
+
+- Extended `RequestAppAudioHandlingParams` by adding `audioDeviceSelectionChangedCallback` for speaker selection updates
+
+### Patches
+
+- Added `meetingStage` and `settings` framecontexts to `clipboard`.
+- Fixed more strictNullChecks violations
+- Added additional telemetry to `App`, `Dialog`, `GeoLocation`, `Location`, `Navigation`, `Pages`, and `Tasks` capabilities
+- Created new `MessageRequest` interface with required properties to enhance type-safety
+- Fixed more violations of strictNullChecks
+- Added telemetry to `barcode`, `calendar`, `call`, `clipboard`, `mail`, `marketplace` and `media` capabilities
+- Fixed strictNullChecks violations in `media.ts`, `mediaUtil.ts`, and other files
+- Fixed calls to `chat.openChat` and `chat.openGroupChat` when only a single user is specified
+- Fixed more `strictNullChecks` violations
```

Comment:
Would be good to remove lines 18 and 25 now that line 23 was updated

Reply:
Planning to do that as part of a bigger change (need to merge in Aaron's fix here https://github.com/OfficeDev/microsoft-teams-library-js/pull/2070#pullrequestreview-1758615669)